src/catalog/data/selectors.js

```diff
@@ -0,0 +1,3 @@
+import { STORE_NAMES } from '../../common/constants';
+
+export default (state) => ({ ...state[STORE_NAMES.CATALOG] });
```

Comment:
Are you using the spread operator here to make a copy so that the original state isn't affected? I don't think it's really needed.